### ISSUE TITLE
Modifications gone missing

### ISSUE THREAD

**Issue by kimrutherford** [pombase/website#943]
The modification appear in the summary view but the residue column has gone missing from the detailed view.  I'm working on this now.

**Comment by kimrutherford** [comment]
I was confused.  There was never a residues column for phenotypes.  Do we want one?  Or should the residue appear beside the extensions like it does in the summary?

  rad26Δ  **affecting** mrc1  **modified residue** T645

**Comment by ValWood** [comment]
It can be in the extensions column as you suggest.
The reason it isn't there is that it was only added recently, and we forgot about the downloads file ;)

**Comment by kimrutherford** [comment]
Just to check before I change it: should it be beside the extensions in the modifications table as well as the phenotypes?  That would be consistent.

**Comment by kimrutherford** [comment]
The modifications table is currently ordered by the residue so it looks quite tidy.  It's going to look worse if the residue moves beside the extension.

![modification-table-1](https://user-images.githubusercontent.com/90474/45285647-c9e72e80-b537-11e8-87d5-457e9cac5675.png)

**Comment by ValWood** [comment]
I think it could be in the extension column because it is a type of extension ...perhaps? to save changing the format.... but wait for @mah11

**Comment by mah11** [comment]
I vote for

- no change to the modification display, and
- for phenotypes, put residues with the rest of the extensions.

This means the residue display won't be identical between phenotype and modification, but I think that's OK because they are different annotation types, and the relative importance of residue info is different.

**Comment by ValWood** [comment]
OK, I completely misunderstood what this ticket was about.
But do we need to alter the PHAF files in any way to cope with recent changes to add residues?

**Comment by mah11** [comment]
This is a website ticket about the display of residue details, so the question isn't about PHAF files. That was covered in https://github.com/pombase/pombase-chado/issues/682

**Comment by kimrutherford** [comment]
Yep.  The residues are successfully stored in Chado and are displayed in the summary.  But they're missing from the detailed phenotype annotation view.  I'll fix that tomorrow now we have a plan.

**Comment by kimrutherford** [comment]
This is fixed for the morning.  I'll check and close the issue then.
